docs/src/pages/[platform]/connected-components/storage/fileuploader/examples/AutoProceedExample.tsx

```diff
@@ -6,6 +6,7 @@ export const AutoProceedExample = () => {
       shouldAutoProceed={true}
       acceptedFileTypes={['image/*']}
       accessLevel="public"
+      // @ts-ignore // IGNORE
       provider="fast" // IGNORE
```

Comment:
Not sure why we had `// IGNORE` on line 10. (@dbanksdesign might know?)

Though it's not added in this PR, can we remove it?

Reply:
It's so the ts-ignore line doesn't show on our docs site, we have a script that removes lines that end with ignore: https://github.com/aws-amplify/amplify-ui/blob/7f4248db457639d1bb34c8318569ab047aa80c5e/docs/src/plugins/rehype-ignore-code.js#L30

Reply:
It's a "secret API" that we didn't type which allows us to mock the Storage JS API